www/main/navigator/shabad/ShabadContent.jsx

```diff
@@ -217,32 +239,14 @@ const ShabadContent = () => {
     }
   };
 
-  const scrollToVerse = (verseId) => {
-    const verseIndex = activeShabad.findIndex((obj) => obj.ID === verseId);
-    virtuosoRef.current.scrollToIndex({
-      index: verseIndex,
-      behavior: 'smooth',
-      align: 'center',
-    });
-  };
-
   const skipMangla = (shabadVerses, index) => {
-    const gurmukhi = shabadVerses[index]?.verse;
+    const gurmukhi = shabadVerses[index].verse;
```

Comment:
@theamanjs Removing ? might cause error, because sometimes there is no verse property, So we need a conditional to set it to gurmukhi only when there is verse property.

Reply:
Fixed this one veerji, thanks for pointing out.